plenoptic/simulate/models/naive.py

```diff
@@ -28,4 +38,180 @@ def forward(self, img):
             a clone of the input image
 
         """
-        return img.clone()
+        y = 1 * img
```

Comment:
why use this instead of `img.clone()`?

Reply:
This way if we ever need to chain this into something the gradient can flow undisturbed.

Reply:
beautiful